src/components/SEP/Proposals/SEPAssignedReviewersTable.tsx

```diff
@@ -117,26 +131,28 @@ const SEPAssignedReviewersTable: React.FC<SEPAssignedReviewersTableProps> = ({
         }
         actions={[
           (rowData) => ({
-            icon:
-              rowData.review?.status === ReviewStatus.DRAFT
-                ? () => <RateReviewIcon />
-                : () => <Visibility />,
+            icon: isDraftStatus(rowData?.review?.status)
+              ? () => <RateReviewIcon data-cy="grade-proposal-icon" />
+              : () => <Visibility data-cy="view-proposal-details-icon" />,
             onClick: () => {
               if (!rowData.review) {
                 return;
               }
 
-              setEditReviewID(rowData.review.id);
+              setUrlQueryParams({
+                modalTab: isDraftStatus(rowData?.review?.status)
+                  ? reviewProposalTabNames.indexOf('Grade')
+                  : reviewProposalTabNames.indexOf('Proposal information'),
```

Comment:
You could have a `cons` for the tab names to avoid typos because I see the text is reused in multiple places. Maybe even put in some shared place because I see the same names is also used in `ProposalTableReviewer.tsx`